src/app/zones/views/ZoneCreateView.vue

```diff
@@ -41,17 +41,45 @@
         <KCard>
           <template #body>
             <div class="form">
-              <div>
-                <h1 class="form-title">
-                  {{ t('zones.routes.create.title') }}
-                </h1>
+              <div class="form-header">
+                <div>
+                  <h1 class="form-title">
+                    <span class="text-gradient">{{ t('zones.form.title') }}</span>
+                  </h1>
+
+                  <p v-if="t('zones.form.description') !== ' '">
```

Comment:
Would `''` and a length check work here?

Reply:
Yep, still the case:

```
Error: [@formatjs/intl Error MISSING_TRANSLATION] Missing message: "zones.form.section.name.description" for locale "en-us", using id as fallback.
```

Reply:
Hmm thats weird we do an `undefined` check for this, oh this one is coming directly from formatjs.

Umm ok. I'd like to be able to specify `''` when we need to if possible. I'd actually like to use the error config that was added to `@kong-ui-public/i18n` a while back, oh and actually a couple of other things now come to think of it. I'll make an issue

Thanks for trying it out tho!